test/TestSqliteInstance.cpp

```diff
@@ -317,11 +416,19 @@ TEST(TestTransaction, TestGetInserId) {
 
     int64_t id = 0;
     ASSERT_EQ(true, instance->getInsertId(id));
-    printf("insertId = %lld\n", id);
+    printf("insertId = %ld\n", id);
```

Comment:
## suggestion

You can use standard cross-platform placeholders such as PRId64